### ISSUE TITLE
Polar plots have gap at antemeridian

### ISSUE THREAD

**Issue by xylar** [MPAS-Dev/MPAS-Analysis#429]
Polar plots of sea-ice and iceberg data have a wedge between -179.75 and 179.75.  This could be fixed by adding one more column to the data set so the start and finish are the same.

![image](https://user-images.githubusercontent.com/4179064/42064036-d086361c-7af1-11e8-80ed-c48d4314739c.png)

**Comment by milenaveneziani** [comment]
There is a way to add a cycle to the lon variable, before plotting with basemap:
```
from mpl_toolkits.basemap import Basemap, addcyclic
...
[fld, lon] = addcyclic(fld, lon)
[lon, lat] = np.meshgrid(lon, lat)
```
I have used this before and it worked for me.

**Comment by xylar** [comment]
Thank you both. I’ve implemented this and will submit a PR tomorrow.

**Comment by xylar** [comment]
Fixed by #430
